pkg/machines/components/diskAdd.jsx

```diff
@@ -208,6 +209,49 @@ const PoolRow = ({ idPrefix, onValueChanged, storagePoolName, vmStoragePools })
     );
 };
 
+class PerformanceOptions extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { expanded: false };
+    }
+
+    render() {
+        const cacheModes = ['default', 'none', 'writethrough', 'writeback', 'directsync', 'unsafe'];
```

Comment:
Any way not to hard code those/retrieve them from some spec? Otherwise it's fine.

Reply:
Ah ya, that's alright then.